src/app/services/sortFunctions/sort.service.ts

```diff
@@ -0,0 +1,47 @@
+import {Injectable} from '@angular/core';
+import {ISelected} from "./iselected";
+import {Movie} from "../../model/movie/movie";
+
+
+@Injectable({
+  providedIn: 'root'
+})
+export class SortService {
+  private _sortFields : ISelected[] = [
+    {
+      title: "Votes",
+      value: (a: Movie, b: Movie): number => a.vote_average - b.vote_average
+    },
+    {
+      title: "Release date",
+      value: (a: Movie, b: Movie): number => {
+        if (a.release_date > b.release_date) {
```

Comment:
this whole logic with if else can be removed if you convert release_date to a number and do a - b